### PR TITLE
WINDUPRULE-482: groovy rule for a removed class

### PR DESCRIPTION
**Pull request by mmelko**

### CODE REVIEW

File: rules-reviewed/camel3/camel2/tests/data/xml-moved-components/context.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+git git<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jonathanvila) [inline rules-reviewed/camel3/camel2/tests/data/xml-moved-components/context.xml:1]: seems this git git text was not put on the place you wanted ?

**Comment by jonathanvila** [comment]
@mmelko please could you review the conflicts ?

**Comment by mmelko** [comment]
done, thanks!

File: rules-reviewed/camel3/camel2/tests/data/xml-moved-components/context.xml (hunk 1 above)
Comment (mmelko) [reply rules-reviewed/camel3/camel2/tests/data/xml-moved-components/context.xml]: omg ..

**Comment by mmelko** [comment]
Hi @jonathanvila , I've added locally another ~4 rules into same file .. may I rebase this ? or wait till this will be merged and create new PR

File: rules-reviewed/camel3/camel2/java-generic-information.windup.groovy [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import org.jboss.windup.reporting.config.Hint
+import org.jboss.windup.reporting.config.Link
+import org.jboss.windup.reporting.model.QuickfixType
+import org.jboss.windup.reporting.quickfix.Quickfix
+import org.jboss.windup.rules.apps.java.condition.JavaClass
+
+final String MOVED_FROM = "org.apache.camel.ThreadPoolRejectedPolicy"
+final String MOVED_TO = "org.apache.camel.util.concurrent.ThreadPoolRejectedPolicy"
+
+Hint createMovedClassHint(String moved, String movedTo, String addInfo) {
```
Comment (jonathanvila) [inline rules-reviewed/camel3/camel2/java-generic-information.windup.groovy:17]: I like this clean approach :+1:

**Comment by jonathanvila** [comment]
@mmelko this is confusing for me. The Jira ticket WINDUPRULE-482 does not correspond to this scope , and also it has linked another PR different than this.
Which jira ticket this PR belongs to ?

**Pull request merged**